Sourcery/Generating/Template/JavaScript/JavaScriptTemplate.swift

```diff
@@ -20,11 +20,34 @@ final class JavaScriptTemplate: Template {
         let path = Bundle(for: JavaScriptTemplate.self).path(forResource: "ejsbundle", ofType: "js")!
         let ejs = try String(contentsOfFile: path, encoding: .utf8)
 
+        var errorContent = ""
```

Comment:
That would work, but the error is only printed in the console and only when Sourcery is not running in watch mode. I used this method because the error is visible in the generated code.

How would the error be visible to the user?